python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -331,6 +333,9 @@ def __init__(self, config: BaseGaapFluxConfig, name, schema, logName=None) -> No
             flagName = self.ConfigClass._getGaapResultName(scalingFactor, "flag_gaussianization")
             flagDefs.add(flagName, "PSF Gaussianization failed when trying to scale by this factor.")
 
+        self.logName = logName
+        self.log = lsst.log.getLogger(self.getLogName())  # getLogName is inherited from the plugin classes.
```

Comment:
No new python code should be using `lsst.log`. `meas_base` does not.  Use `logging.getLogger()`.

What log name does this end up as in practice? Should the `logName` default to something that has `gaap` in it?

Reply:
The `BaseMeasurementTask` seems to assign the `logName`, with the plugin name included. https://github.com/lsst/meas_base/blob/c63f6ee5de4496055ae9f4a2d7b9beb5fae67e73/python/lsst/meas/base/baseMeasurement.py#L272

Reply:
The log name will end up as `measurement.ext_gaap_GaapFlux` in practice. Now, unlike `lsst.log`, Python's default logging method doesn't seem to being the message with the logger name. So running the `testFail` method from the command line produces

```
RROR: Unable to calculate psf matching kernel
Failed to solve for PSF matching kernel in GAaP for (100.000000, 670.000000): Problematic scaling factors = 100.0 Errors: Exception('Unable to determine kernel sum; 0 candidates')
ERROR: Unable to calculate psf matching kernel
Failed to solve for PSF matching kernel in GAaP for (100.000000, 870.000000): Problematic scaling factors = 100.0 Errors: Exception('Unable to determine kernel sum; 0 candidates')
ERROR: Unable to calculate psf matching kernel
Failed to solve for PSF matching kernel in GAaP for (-10.000000, -20.000000): Problematic scaling factors = 100.0 Errors: Exception('Unable to determine kernel sum; 0 candidates')
```
unlike the [previous comment](https://github.com/lsst/meas_extensions_gaap/pull/19#discussion_r690851862).

Reply:
I think that’s actually ok.  See discussion on this PR: https://github.com/lsst/meas_extensions_piff/pull/4

Reply:
Yes, this is default python logging and we configure it when pipetask or command line tasks run. If you want to test that a log message is actually being issued you should use `self.assertLogs` and with that you can check that an ERROR is issued and also what the message contains. Testing that an ERROR log message is issued might well be a good idea for this ticket.

Reply:
Done.